test/compilable/ddoc17697.d

```diff
@@ -0,0 +1,15 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o-
+// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh 17697
+
+/***
+ * See:
+ *    http://www.fooa.com/test1
```

Comment:
And even if there is, I think it would still be a good idea to test the combination of these two features.